ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -460,5 +460,19 @@
     "CaseEventID": "draftCMO",
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "actionCMO",
+    "UserRole": "caseworker-publiclaw-judiciary",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "actionCMO",
+    "UserRole": "caseworker-publiclaw-courtadmin",
```

Comment:
I believe with this Gatekeeper will also gain ability to action CMO. It is not something story mentioned. Is that acceptable? If not is there a way to override permission for gatekeeper to take it await from them?

Reply:
this is fine following y discussions with the BAs and we also have a [Jira](https://tools.hmcts.net/jira/browse/FPLA-840) to fix the roles.